components/filemanager/events.md

```diff
@@ -413,14 +413,14 @@ The `ViewChanged` event fires when the user toggles between the [two FileManager
 
         if (parentDirectory != null)
         {
-            // simulate add in file system
+            // Simulate add to the file system
```

Comment:
I recommend using ending punctuation consistently. Either all code comments should end with a period (or the semi-colon in some of the instances), or none.